### PR TITLE
Fix several pylint warnings.

### PR DESCRIPTION
**Pull request by LaszloLango**
Fixed all of the remaining warnings in 'build.py', 'run-tests.py' and 'check-license.py'.

Related issue: #1600

JerryScript-DCO-1.0-Signed-off-by: László Langó llango.u-szeged@partner.samsung.com

### CODE REVIEW

File: tools/unicode_ranges.py [hunk 1]
```diff
@@ -229,7 +231,7 @@ def ranges(i):
     :return: List of ranges.
     """
 
-    for a, b in itertools.groupby(enumerate(i), lambda (x, y): y - x):
+    for _, b in itertools.groupby(enumerate(i), lambda (x, y): y - x):
```
Comment (zherczeg) [inline tools/unicode_ranges.py]: underscore variable?

Comment (robertsipka) [inline tools/unicode_ranges.py]: It's correct, a throwaway variable name to indicate that part of the result is being ignored.

Comment (robertsipka) [inline tools/unicode_ranges.py]: But, I fixed these pylint warnings (unicode related warnings) in the #1623 PR.

Comment (LaszloLango) [inline tools/unicode_ranges.py]: I thought that patch landed already :)

**Comment by robertsipka** [comment]
LGTM (informally)

**Pull request merged**